website/content/en/docs/about/concepts.md

```diff
@@ -24,6 +24,11 @@ A **metric** event represents a numerical operation performed on a time series.
 
 {{< jump "/docs/about/under-the-hood/architecture/data-model/metric" >}}
 
+### Traces
+
+A **trace** event can be thought of as a special kind of log event. Currently Vector only supports trace events for the `datadog_agent` source, and the `datadog_traces` sink.
```

Comment:
technically there's some transforms that support, not sure what you wanted to list?

Reply:
would that be Sample and Remap ?

Reply:
IIRC, yes 😅

Reply:
🤔 My intent with that line was to better communicate _which_ components we already have support for that in 😅

Reply:
Yeah, balance between knowing for sure that list is going to get out of date and being more useful for the users.

Ideally the site would be rendering what event types each component can handle, it used to but doesn't anymore.

Reply:
Perhaps a medium could be something like:

"At this writing, the components that support Trace events are <...>". If you're interested in using traces with Vector and have a component that doesn't yet support them, please open an issue so we can better prioritize adding support."